app/controllers/users.js

```diff
@@ -54,14 +54,18 @@ module.exports = {
         }
         if (bcrypt.compareSync(password, user.password)) {
           logger.info(`User ${email} authenticated.`);
-          const token = jwt.sign(JSON.stringify(user), config.secret);
-          return res.status(200).json({ token });
+          const token = jwt.sign(JSON.parse(JSON.stringify(user)), config.secret, {
+            expiresIn: config.expirationInSeconds
+          });
+          return res
+            .status(200)
+            .json({ token, expirationDate: Math.floor(Date.now() / 1000) + config.expirationInSeconds });
```

Comment:
I strongly recommend to use some package for dates and times logic, like moment.js

Reply:
added